src/test/java/JSONParserTest.java

```diff
@@ -0,0 +1,75 @@
+import Group15.Api.Exercise;
+import Group15.JSONParser;
+import Group15.Workout;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class JSONParserTest
+{
+    // Can hold objects between tests:
+    private String exercisesFilePath;
+    private String workoutsFilePath;
+    private List<Exercise> exercises;
+    private List<Workout> workouts;
+
+    private void initExercises()
+    {
+        exercises = new ArrayList<>();
+
+        Exercise pushup = new Exercise("Pushup", "Push your body up and down", "Strength", "Chest", "None", "Easy", "None");
+        Exercise situp = new Exercise("Situp", "Sit up and down", "Strength", "Abs", "None", "Easy", "None");
+        exercises.add(pushup);
+        exercises.add(situp);
+    }
+
+    private void initWorkouts()
+    {
+        workouts = new ArrayList<>();
+
+        Workout workout = new Workout();
+        for (Exercise exercise : exercises)
+        {
+            workout.addExercise(exercise);
+        }
+        workouts.add(workout);
+        workout = new Workout();
```

Comment:
Is there a reason you are creating a new instance after populating the workout object?

Reply:
Because it is a pointer